frontend/src/pages/Profile/UserCourses/UserCourses.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { useState } from 'react';
+import { Divider, Input } from '@mui/material';
+import { Star as StarIcon, Search as SearchIcon } from '@mui/icons-material';
+import InputAdornment from '@material-ui/core/InputAdornment';
+
+import { Course } from 'types/course';
+import isCourseCompleted from 'utils/helpers/isCourseCompleted';
+
+import CourseMaterialInfoContainer from 'pages/Profile/UserCourses/CourseMaterialInfoContainer';
+import {
+  CoursesBox,
+  CoursesList,
+  CoursesListItem,
+  CourseTitle,
+  Title,
+  MaterialsList,
+} from './styled';
+
+interface CoursesProps {
+  courses?: Array<Course>;
+}
+
+const UserCourses: React.FC<CoursesProps> = ({ courses }) => {
+  const [searchCourse, setSearchCourse] = useState('');
+  console.log(courses);
+  return courses?.length === 0 ? (
+    <div>No courses chosen</div>
+  ) : (
```

Comment:
```suggestion
  return courses?.length ? (
    <div>No courses chosen</div>
  ) : (
```

Reply:
In case I write courses?.length this statement is true and block with courses is shown, even though courses is an empty array. But if I write courses?.length === 0 it works as it should.